python/ray/_private/runtime_env/conda_utils.py

```diff
@@ -173,6 +173,33 @@ def get_conda_env_list() -> list:
     return envs
 
 
+def get_conda_info_json() -> dict:
```

Comment:
Can you add a link or an example of the return value?

Reply:
done